CONTRIBUTING.md

```diff
@@ -234,12 +216,20 @@ When submitting a component to the community, consider breaking it down into sep
 * **Second PR** should include the concrete implementation of the component. If the
   size of this PR is larger than the recommended size consider splitting it in
   multiple PRs.
-* **Last PR** should mark the new component as `Alpha` stability and add it to the `cmd/otelcontribcol`
-  binary by updating the `cmd/otelcontribcol/builder-config.yaml` file and running `make genotelcontribcol`.
-  The component's tests must also be added as a part of its respective `component_type_tests.go` file in the `cmd/otelcontribcol` directory.
-  The component must be enabled only after sufficient testing and only when it meets [`Alpha` stability requirements](https://github.com/open-telemetry/opentelemetry-collector#alpha).
-* Once a new component has been added to the executable, please add the component
-  to the [OpenTelemetry.io registry](https://github.com/open-telemetry/opentelemetry.io#adding-a-project-to-the-opentelemetry-registry).
+* **Last PR** should mark the new component as `Alpha` stability.
+  * Update its `metadata.yaml` file.
+    * Mark the stability as `alpha`
+    * Add `contrib` to the list of distributions
+  * Add it to the `cmd/otelcontribcol` binary by updating the `cmd/otelcontribcol/builder-config.yaml` file.
+  * Please also run:
+    - `make generate`
+    - `make genotelcontribcol`
+  * The component's tests must also be added as a part of its respective `component_type_tests.go` file in the `cmd/otelcontribcol` directory.
+  * The component must be enabled only after sufficient testing and only when it meets [`Alpha` stability requirements](https://github.com/open-telemetry/opentelemetry-collector#alpha).
+  * Please also submit a PR to the [OpenTelemetry Collector Releases](https://github.com/open-telemetry/opentelemetry-collector-releases) repository to include your component in future releases of the OpenTelemetry Collector `contrib` distribution.
```

Comment:
Maybe we should add a note here that this PR should be created after upgrading the stability (or in parallel), maybe this point should be at the top level instead of being underneath the `Last PR` section, feedback is welcome here.

Reply:
I would make it top level and make it clear that it happens after alpha stability has been reached (I think it's important that we make sure the component has reached the alpha quality level before we add to contrib)

Reply:
I've moved it to the top level and hopefully made it more clear that it's **after** `Alpha`. Let me know if it's what you were thinking 👍

Reply:
Awesome, thanks!!